spec/util/resources_builder_spec.rb

```diff
@@ -9,20 +9,20 @@ class TestUselessResource < LedgerSync::Resource
     end
 
     class TestGrandchildResource < LedgerSync::Resource
-      reference :useless_child,  TestResource::TestUselessResource
+      has_one :useless_child,  TestResource::TestUselessResource
```

Comment:
We need some `has_many` tests here too.

Reply:
Done. In parent resource I change `has_one child` and `has_one child2` as `has_many :children`.